### PR TITLE
Adding dynamodb scaler documentation

### PR DESCRIPTION
**Pull request by samueleresca**
Documentation regarding  the AWS DynamoDB scaler: [https://github.com/kedacore/keda/pull/2486](https://github.com/kedacore/keda/pull/2486)
### Checklist

- [X] Commits are signed with Developer Certificate of Origin (DCO)

Fixes https://github.com/kedacore/keda/issues/2482

### CODE REVIEW

File: content/docs/2.6/scalers/aws-dynamodb.md
Comment (arschles) [inline content/docs/2.6/scalers/aws-dynamodb.md:11]:
```suggestion
This specification describes the scaler AWS DynamoDB scaler. This scaler uses a specified DynamoDB query to determine if and when to scale a given workload.
```

**Review by arschles — CHANGES_REQUESTED** [review]

File: content/docs/2.6/scalers/aws-dynamodb.md
Comment (arschles) [inline content/docs/2.6/scalers/aws-dynamodb.md:11]:
```suggestion
This specification describes the AWS DynamoDB scaler. This scaler uses a specified DynamoDB query to determine if and when to scale a given workload.
```

**Review by arschles — CHANGES_REQUESTED** [review]

**Review by arschles — APPROVED** [review]

File: content/docs/2.6/scalers/aws-dynamodb.md
Comment (tomkerkhove) [inline content/docs/2.6/scalers/aws-dynamodb.md:1]: We have already shipped v2.6 - Would you mind moving it to the v2.7 folder and update the availability here please?

File: content/docs/2.6/scalers/aws-dynamodb.md
Comment (tomkerkhove) [inline content/docs/2.6/scalers/aws-dynamodb.md:20]: Is table name the same as the resource name or?

**Review by tomkerkhove — COMMENTED** [review]
Did a first rough review but things look good AFAIK, apart from a few topics

**Review by tomkerkhove — APPROVED** [review]

File: content/docs/2.6/scalers/aws-dynamodb.md
Comment (samueleresca) [reply content/docs/2.6/scalers/aws-dynamodb.md]: The tableName and region identify univocally the dynamoDB endpoint to retrieve the data. Therefore, I should have chosen a better value for `tableName`, such as `myTableName`. I'm proceeding by updating that in this PR.

**Review by arschles — APPROVED** [review]

**Comment by tomkerkhove** [comment]
LGTM, merging once conflicts are fixed and feature PR is merged.

**Review by JorTurFer — CHANGES_REQUESTED** [review]
LGTM!
Only small nit

**Review by JorTurFer — APPROVED** [review]
LGTM!
Thanks for this! ❤️

**Pull request merged**
